vars.py

```diff
@@ -1 +1 @@
-token='YOUR_TOKEN_HERE'
+token='YOUR-TOKEN-HERE'
```

Comment:
We'll want to add this to the gitignore as well, so that when we put in our new tokens we don't commit them, the vars file (after adding the debug flag maybe(?)) should be relatively sparse in changes compared to other ones, so manually needing to commit it and having it automatically ignored is probably a good idea so we don't constantly fuck up and commit our tokens.